### DISCUSSION TITLE
q-carousel-slide and v-for directive

### DISCUSSION THREAD

**Discussion by alessioguidi** [quasarframework/quasar#12260] [General - Components / Directives / etc]
I have a problem with q-carousel-slide and v-for.
Slide selection doesn't seem to work. It does not change the v-model value

See codepen https://codepen.io/alessioguidi/pen/LYOpZOO

Where am I doing wrong?

Thanks

**Comment by MilosPaunovic** [comment]
```js
setup () {
  return {
    carousel: ref("1"),
    // ...
  }
}
```

**Comment by alessioguidi** [comment]
Hi Milos,
how can i use ref in case of v-model on element of an object?
v-model="jx.v.o2q_carousel21.valore"

see updated codepen  https://codepen.io/alessioguidi/pen/LYOpZOO

thanks

Il giorno dom 30 gen 2022 alle ore 13:11 Milos Paunovic <
***@***.***> ha scritto:

**Comment by MilosPaunovic** [comment]
Isn't this now another question?
The answer should be that `QCarousel` requires reactive property for its model.
